apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1961,6 +1965,390 @@ static void libc_stdio_setvbuf_tc(void)
 #endif
 
 /**
+* @testcase         libc_stdio_meminstream_tc
+* @brief            Initializes a stream for use with a fixed-size memory buffer
+* @scenario         Initializes a stream for use with a fixed-size memory buffer
+* @apicovered       lib_meminstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_meminstream_tc(void)
+{
+	FAR char buf[BUFLEN];
+
+	struct lib_meminstream_s meminstream;
+
+	lib_meminstream((FAR struct lib_meminstream_s *)&meminstream, buf, BUFLEN);
+	TC_ASSERT_EQ("lib_meminstream", meminstream.buffer, (FAR char *)(buf));
+	TC_ASSERT_EQ("lib_meminstream", meminstream.buflen, BUFLEN);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_memoutstream_tc
+* @brief            Initializes a stream for use with a fixed-size memory buffer
+* @scenario         Initializes a stream for use with a fixed-size memory buffer
+* @apicovered       lib_memoutstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_memoutstream_tc(void)
+{
+	FAR char buf[BUFLEN];
+
+	struct lib_memoutstream_s memoutstream;
+
+	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buf, BUFLEN);
+	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer, (FAR char *)(buf));
+	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buflen, (BUFLEN - 1));	/* Save space for null terminator, hence checing with (BUFLEN-1)*/
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_memsistream_tc
+* @brief            Initializes a stream for use with a fixed-size memory buffer
+* @scenario         Initializes a stream for use with a fixed-size memory buffer
+* @apicovered       lib_memsistream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_memsistream_tc(void)
+{
+	FAR char buf[BUFLEN];
+
+	struct lib_meminstream_s meminstream;
+	struct lib_memsistream_s memsistream;
+
+	lib_meminstream((FAR struct lib_meminstream_s *)&meminstream, buf, BUFLEN);
+	TC_ASSERT_EQ("lib_meminstream", meminstream.buffer, (FAR char *)(buf));
+
+	lib_memsistream((FAR struct lib_memsistream_s *)&memsistream, buf + 2, BUFLEN);
+	TC_ASSERT_EQ("lib_memsistream", (memsistream.buffer - (FAR char *)(buf)), 2);
```

Comment:
what means 2? just any number?

Reply:
Yes , just to seek through that stream and check if its seeking correctly